### ISSUE TITLE
2.7.0 doesn't work with ldap secured password on Windows 

### ISSUE THREAD

**Issue by madumro** [adobe-apiplatform/user-sync.py#759]
**Description**
When storing ldap password in Windows Credential Manager, using 'secure_password_key: ldap_password', UST 2.7.0 throws the error:
CRITICAL main - ldap configuration: No value in secure storage for user "CBART\Administrator", key "ldap_password"

**Steps to reproduce**

- in 'connector-ldap.yml' file, use 'secure_password_key: ldap_password' instead of plain password
- add ldap_password in Windows Credential Manager (Generic credentials)

**Expected behavior**
UST 2.7.0 should find the password in Windows Credential Manager.

**Environment**

UST version: 2.7.0
OS type and version: Windows Server 2016

**Comment by jarey14** [comment]
+1
I also experienced the same problem after migrating to UST 2.7.1 (from a really old version of UST that uses user-sync.pex).  v2.7.1 just will not pull anything from the Windows Credential keys.  I tried not using the ldap_password key and instead hard-coding the LDAP password into the connector-ldap.yml file, and that was able to get an LDAP connection, but then I subsequently received the same "No value in secure storage" regarding the umapi_api_key on the next line of output.  To me that suggests OST 2.7.x currently can't access any keys in Cred Manager.  I've rolled back to version 2.6.6, which works well with no changes to my config.

OS:  Windows Server 2019.
